### PR TITLE
Fix VFs configuration for InfiniBand NICs

### PR DESCRIPTION
**Pull request by e0ne**
Closes: #274

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -299,7 +299,13 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 			// for userspace drivers like vfio we configure the vf mac using the kernel nic mac address
 			// before we switch to the userspace driver
 			if yes, d := hasDriver(addr); yes && !sriovnetworkv1.StringInArray(d, DpdkDrivers) {
-				if strings.EqualFold(iface.LinkType, "IB") {
+				// LinkType is an optional field. Let's fallback to current link type
+				// if nothing is specified in the SriovNodePolicy
+				linkType := ifaceStatus.LinkType
```
Comment (SchSeba) [inline pkg/utils/utils.go:304]: I think here you want to put the sriov policy?

Comment (e0ne) [reply pkg/utils/utils.go:304]: thanks for the pointing me on it. will fix this typo asap.

**Comment by e0ne** [comment]
Go / test (pull_request) failure is not related to this change

**Comment by e0ne** [comment]
@pliurh   @zshi-redhat could you please run e2e ci?

**Comment by SchSeba** [comment]
looks like there is a problem with the controller-gen

**Comment by e0ne** [comment]
> looks like there is a problem with the controller-gen

Is seems to be a golang issue witch is fixed in pr #277

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**